addons/io_hubs_addon/debugger.py

```diff
@@ -53,7 +53,7 @@ def is_room_set(context):
 class HubsUpdateRoomOperator(bpy.types.Operator):
     bl_idname = "hubs_scene.update_room"
     bl_label = "View Scene"
-    bl_description = "Update room"
+    bl_description = "Updates the currently opened room scene with the Blender scene"
```

Comment:
This should use the description classmethod so that the text can be changed depending on whether the scene is being updated or spawned as an object.

Something like this:
```
@classmethod
def description(cls, context, properties):
    is_scene_update = context.scene.hubs_scene_debugger_room_create_prefs.debugLocalScene
    if hubs_session.is_alive():
        room_params = hubs_session.room_params
        is_scene_update = "debugLocalScene" in room_params

    if is_scene_update:
        return "Updates the currently opened room scene with the Blender scene"
    else:
        return "Spawns the Blender scene in the currently opened room as an object"
```
https://docs.blender.org/api/current/bpy.types.Operator.html#bpy.types.Operator.description

Reply:
Oh cool, I dind't know this